pages/stats/challenges.tsx

```diff
@@ -0,0 +1,340 @@
+import { GetStaticProps, NextPage } from "next";
```

Comment:
@KingDarBoja  we need to move also this file to pages/explorer/ 

otherwise it should be done :) 👍